spec/integration/plan_spec.rb

```diff
@@ -156,14 +155,28 @@
           }
         }
 
-        with_project(config: project_config) do |project|
-          config_flags = %W[-m #{modulepath} --project #{project.path} --no-host-key-check]
-          expect(Dir.children(project.path)).not_to include('.plan_cache.json')
+        it 'caches plan info when generating types', ssh: true do
+          with_project(config: project_config) do |project|
+            config_flags = %W[-m #{modulepath} --project #{project.path} --no-host-key-check]
+            expect(Dir.children(project.path)).not_to include('.plan_cache.json')
 
-          run_cli(%w[module generate-types] + config_flags)
-          expect(Dir.children(project.path)).to include('.plan_cache.json')
-          cache = JSON.parse(File.read(project.plan_cache_file))
-          expect(cache).to include(fact_info)
+            run_cli(%w[module generate-types] + config_flags)
+            expect(Dir.children(project.path)).to include('.plan_cache.json')
+            cache = JSON.parse(File.read(project.plan_cache_file))
+            expect(cache).to include(fact_info)
+          end
+        end
+
+        it ' clears existing cache with --clear-cache' do
+          with_project(config: project_config) do |project|
+            config_flags = %W[-m #{modulepath} --project #{project.path} --no-host-key-check]
+            run_cli(%w[module generate-types] + config_flags)
+            original_mtime = File.mtime(project.plan_cache_file)
+            run_cli(%w[plan show] + config_flags)
+            expect(original_mtime).to eq(File.mtime(project.plan_cache_file))
```

Comment:
Same as below, the initial `plan show` call is just to ensure that cache was matching behavior from other tests in file. Probably not necessary

Reply:
Yeah, checking mtimes has proven pretty flaky, you can remove this `expect`

Reply:
Now that there's a test in `cli_spec.rb` should I just delete what I did in the integration testing?